### PR TITLE
Update index.js

### PR DESCRIPTION
**Pull request by JakenHerman**
Added hacktoberfest ascii kitten

### CODE REVIEW

File: app/index.js [hunk 1]
```diff
@@ -10,6 +10,23 @@ function checkWater() {
   console.log(orange('Check your pets water and food, make sure its all fine :)'));
 }
 
+var cat_ascii = `
```
Comment (troy0820) [inline app/index.js]: Change this var to `const`.  We have a .eslintrc.json file in this repo to check for this in your editor/test when you commit.  It won't pass travis-ci which is why your PR is labeled with a red X

**Review by troy0820 — CHANGES_REQUESTED** [review]

File: app/index.js [hunk 2]
```diff
@@ -10,6 +10,23 @@ function checkWater() {
...
+            |     |          Happy Hacktoberfest!
+           /| | | |\\
+           \\| | |_|/\\
+           //_// ___/
+               \\_)
+`;
+
+//hacktoberfest special
+console.log(orange(cat_ascii));
+
```
Comment (lynnaloo) [inline app/index.js:29]: 😹

**Review by JakenHerman — COMMENTED** [review]
changed `var` to `const`

**Review by JakenHerman — COMMENTED** [review]
changed `cat_ascii` to `catAscii` to pass camelcase test on Travis

**Review by JakenHerman — COMMENTED** [review]
forgot to change the const name in console.log on line 28. What a mess I am today.

**Pull request merged**